src/Entity/ProcurationProxy.php

```diff
@@ -666,4 +700,68 @@ public function setProxiesCount(int $proxiesCount): void
     {
         $this->proxiesCount = $proxiesCount;
     }
+
+    public function isFrenchRequestAvailable(): bool
+    {
+        return $this->frenchRequestAvailable;
+    }
+
+    public function setFrenchRequestAvailable(bool $frenchRequestAvailable): void
```

Comment:
IMO I think we don't need to open these control methods, just process and unprocess are public for change this state, WDYT?

Reply:
yep good idea :+1: